prototype_source/vulkan_workflow.rst

```diff
@@ -0,0 +1,187 @@
+PyTorch Vulkan Backend User Workflow
+====================================
+
+**Author**: `Ivan Kobzarev <https://github.com/IvanKobzarev>`_
+
+Introduction
+------------
+PyTorch 1.7 supports the ability to run model inference on GPUs that support the Vulkan graphics and compute API. The primary target devices are mobile GPUs on Android devices. The Vulkan backend can also be used on Linux, Mac, and Windows desktop builds to use Vulkan devices like Intel integrated GPUs. This feature is in the prototype stage and is subject to change.
+
+Building PyTorch with Vulkan backend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Vulkan backend is not included by default. The main switch to include Vulkan backend is cmake option ``USE_VULKAN``, that can be set by environment variable ``USE_VULKAN``.
+
+To use PyTorch with Vulkan backend, we need to build it from source with additional settings. Checkout the PyTorch source code from GitHub master branch.
+
+Vulkan wrapper
+^^^^^^^^^^^^^
+
+By default, Vulkan library will be loaded at runtime using the vulkan_wrapper library. If you specify the environment variable ``USE_VULKAN_WRAPPER=0`` libvulkan will be linked directly.
+
+Desktop build
+^^^^^^^^^^^^^
+
+Download VulkanSDK from https://vulkan.lunarg.com/sdk/home and set environment variable ``VULKAN_SDK``
+
+For Linux:
+
+::
+
+    cd PYTORCH_ROOT
+    USE_VULKAN=1 USE_VULKAN_SHADERC_RUNTIME=1 USE_VULKAN_WRAPPER=0 python setup.py install
```

Comment:
`install` should probably be `develop` as otherwise running the Python script to save pretrained mobilenet_v2 to a file causes error (details below).